### DISCUSSION TITLE
`newToolbars`: Give us your thoughts!

### DISCUSSION THREAD

**Discussion by GerardasB** [iTwin/appui#924] [Preview Features]
The `newToolbars` feature replaces existing `Toolbar` components with `iTwinUI` based toolbars. This change aims to modernize the toolbar's appearance and functionality, aligning it with the latest design standards and accessibility requirements.

[Live demo](https://itwin.github.io/appui/storybook/?path=/story/components-toolbarcomposer--action-item&args=newToolbars:!true&globals=resizer:true)

> [!NOTE]  
> Released in [4.11.0](https://github.com/iTwin/appui/releases/tag/release%2F4.11.0)
> Removed in [5.0.0](https://github.com/iTwin/appui/releases/tag/release%2F5.0.0) (new toolbars are currently used by default)

- Slimmer design

  | Before  | After |
  | ------------- | ------------- |
  | ![Screenshot 2024-03-19 at 17 19 50](https://github.com/iTwin/appui/assets/10091419/1ca88591-359a-402b-ab07-8daa9a0c7114) | ![Screenshot 2024-03-19 at 17 21 12](https://github.com/iTwin/appui/assets/10091419/588b13b7-633d-4ea7-996b-694a4c451861)  |

- Single-column toolbar group item
  | Before  | After |
  | ------------- | ------------- |
  |  ![Screenshot 2024-03-19 at 17 55 19](https://github.com/iTwin/appui/assets/10091419/662255e9-cabf-4899-8ece-53658aaeebb2) |  ![Screenshot 2024-03-19 at 17 55 29](https://github.com/iTwin/appui/assets/10091419/91bc0822-d70d-491e-ae24-3e1ca529d834)  |

- Renders expandable submenu instead of a nested menu for toolbar group items
  | Before  | After |
  | ------------- | ------------- |
  | ![Screenshot 2024-03-19 at 17 27 46](https://github.com/iTwin/appui/assets/10091419/48f88300-4e77-4573-ab6a-94814fe9e171) | ![Screenshot 2024-03-19 at 17 27 57](https://github.com/iTwin/appui/assets/10091419/840328c0-34d7-435b-b927-9af6b172dff2)  |

- Styled labels
  | Before  | After |
  | ------------- | ------------- |
  | ![Screenshot 2024-03-19 at 17 28 22](https://github.com/iTwin/appui/assets/10091419/82afa941-a6a6-403a-9c40-63ee611dd13c) | ![Screenshot 2024-03-19 at 17 28 55](https://github.com/iTwin/appui/assets/10091419/616897e9-0751-4678-8ef9-de396c644a4a)  |

## Why is it a preview feature?

- We want to determine any style or behavior inconsistencies before replacing the existing implementation.

**Comment by aruniverse** [comment]
I like the more consistent tooltips with the new toolbars, but not a fan of the dark border of the toolbar itself
